packages/lesswrong/components/editor/EditorFormComponent.tsx

```diff
@@ -275,17 +279,15 @@ export const EditorFormComponent = ({form, formType, formProps, document, name,
     && (collectionName!=="Tags" || formType==="edit");
   
   const actualPlaceholder = (editorHintText || hintText || placeholder);
-  
-  if (!document) return null;
```

Comment:
Could you say a bit about why this check was removed?

Reply:
eslint was complaining that it was used without null-check earlier in the file (seems to be used in a bunch of places).

I wasn't actually sure if it was right to remove it but it's complained the last few times I looked in this file. In previous times I just left it, but figured this time I would force some discussion of it and make a final call.

The type for document is `document: any`, which is maybe not maximally reassuring.

Reply:
Based on the use of `EditorFormComponent` as the control in the `contents` field created by `make_editable`, it does seem like we do sometimes explicitly return `null` (if `checkAccess` fails).

Reply:
I added it back in, and added a comment about it